vllm/config/kv_transfer.py

```diff
@@ -64,6 +64,11 @@ class KVTransferConfig:
     enable_permute_local_kv: bool = False
     """Experiment feature flag to enable HND to NHD KV Transfer"""
 
+    kv_load_retry_policy: Literal["recompute", "fail"] = "recompute"
```

Comment:
Maybe change the name since neither of these involve retrying?
```suggestion
    kv_load_failure_policy: Literal["recompute", "fail"] = "recompute"
```

I guess a separate question is whether we should also retry to get the blocks from the connector. It could make sense to do that (or not do it) in conjunction with either of these.

I'm also wondering (just thinking aloud) whether it would make sense for this to be controlled instead via a "strict decode-only mode" parameter.

Reply:
> I guess a separate question is whether we should also retry to get the blocks from the connector. It could make sense to do that (or not do it) in conjunction with either of these.

I agree, I think we'd need a separate config for this that can be overridden, and let the connectors define their own default behavior (which includes retry policy). It seems out of scope for this PR though

Reply:
@njhill @wseaton, I think a "strict decode-only mode" may be too harsh in many cases. It blocks the option for a router to decide not to disaggregate a request (e.g. due to high cache hit on a decode node)
Also in case of load failures, if say 95% of the tokens were loaded successfully and only a small tail at the end failed, it may make sense to perform some prefill on the decode node (similar to the decision not to disaggregate). This is one of the use-cases for the cache_hit_threshold feature #24520